.taskcluster.yml

```diff
@@ -34,7 +34,8 @@ tasks:
           image: python:3.9
           command: >-
             pip install -r requirements.txt &&
-            python3 manage.py test
+            python3 manage.py test &&
+            python3 manage.py makemigrations --dry-run --check
```

Comment:
Thanks! I deleted my other comment so I could make it here where it belongs, but you are too fast @djmitche...